docs/operating/transaction-gas.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Transaction gas
+proofedDate: na
+iterationBy: na
+includedInSite: true
+approvedBy: na
+comment: 
+---
+
+## TL;DR
+
+Operators: 
+
+- [Configure `PRX_OPERATOR_FEE`](#gas-price-the-operator-fee)
+- [Set up price sources by network](#calculation-configuration)
+- [May reject transactions based on gas price](#minimum-gas-price)
+- [Can set gas to zero for test networks](#zero-gas-price-for-testing)
+- Must configure the Proxy to accept no-fee transactions
```

Comment:
@afalaleev line 18 -- is there any configuration required for POs to accept the no-fee tx or is this all managed automatically by the updated Proxy?

Reply:
Andrey will provide this information